ddp-workspace/projects/ddp-dsm-ui/src/app/scan/scan.component.ts

```diff
@@ -450,4 +450,22 @@ export class ScanComponent implements OnInit {
     }
     return null;
   }
+
+  isNotSixCharacters(): boolean {
+    if(!this.initialScan)
+    {
+      return this.scanPairs.length < 2
+    }
+
+    if(this.scanPairsValue.length === 0) {
+      return true;
+    }
+    for (var pair in this.scanPairsValue) {
```

Comment:
I would suggest using the let keyword (or const) instead of var, as var stores variables in a global object, which later might be problematic.
[Article](https://dev.to/mindninjax/stop-using-var-for-declaring-variables-2p3a) for reference

Reply:
[More about the VAR keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var)

Reply:
Done Diddly Did. I added a message 👍